src/rating/star-rating.js

```diff
@@ -79,6 +79,9 @@ class StarRating extends React.Component<StarRatingPropsT, RatingStateT> {
           $isActive={
             previewIndex !== undefined ? x <= previewIndex : x <= value
           }
+          $isPartialActive={
```

Comment:
I can see the argument for making this an opt-in prop. Happy to add if you like. IMO omitting the prop is simpler, and it's fairly unsurprising default behaviour.

But very happy to make this opt-in if people prefer 🙂